app/views/admin/emails/custom_email.html.haml

```diff
@@ -0,0 +1,60 @@
+.row
+  .col-md-6.col-md-offset-2
+    .panel.panel-default
+      .panel-heading
+        %h3.panel-title
+          Send an email
+      .panel-body
+        %p
+          %b Select/Enter Emails
+        = text_field_tag :to, '', class: 'form-control', id: 'admin_email_to', data: { keys: @keys }
+        %br
+        %p.text-right
+          = link_to 'Send Email', '', class: 'btn btn-primary', id: 'send_mail'
+
+    :javascript
+      $(document).ready(function() {
+        var $select = $('#admin_email_to').selectize({
+           delimiter: ',',
+           persist: true,
+           plugins: ['remove_button'],
+           labelField:  'item',
+           valueField:  'item',
+           sortField:   'item',
+           searchField: 'item',
+           create: function(input) {
+             if (!/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(input)){
+               return {}
+             }
+             return {
+                 item: input
+             }
+           }
+        })
+        var selectizeControl = $select[0].selectize;
+        var tags = $('#admin_email_to').data('keys')
+        for(var i in tags){
```

Comment:
It's best if we name our variables in a way that they convey their meaning.

Reply:
on it.